Bio/codonalign/__init__.py

```diff
@@ -34,32 +34,31 @@ def build(pro_align, nucl_seqs, corr_dict=None, gap_char='-', unknown='X',
 
     Arguments:
      - pro_align  - a protein MultipleSeqAlignment object
-     - nucl_align - an object returned by SeqIO.parse or SeqIO.index
+     - nucl_seqs - an object returned by SeqIO.parse or SeqIO.index
        or a collection of SeqRecord.
      - alphabet   - alphabet for the returned codon alignment
      - corr_dict  - a dict that maps protein id to nucleotide id
      - complete_protein - whether the sequence begins with a start
        codon
-     - frameshift - whether to apply frameshift detection
 
     Return a CodonAlignment object
-
-    >>> from Bio.Alphabet import IUPAC
+    
```

Comment:
There is some trailing white space here which is what the flake8 check on TravisCI is complaining about:

```
$ flake8 Bio/
Bio/codonalign/__init__.py:45:1: W293 blank line contains whitespace
```